Src/PCompiler/PCommandLine/Options/PCheckerOptions.cs

```diff
@@ -70,6 +70,7 @@ internal PCheckerOptions()
             advancedGroup.AddArgument("graph-bug", null, "Output a DGML graph of the iteration that found a bug", typeof(bool));
             advancedGroup.AddArgument("graph", null, "Output a DGML graph of all test iterations whether a bug was found or not", typeof(bool));
             advancedGroup.AddArgument("xml-trace", null, "Specify a filename for XML runtime log output to be written to", typeof(bool));
+            advancedGroup.AddArgument("json-trace", null, "Specify a filename for JSON runtime log output to be written to", typeof(bool));
```

Comment:
Suggestion: make "json-trace" option hidden (and make it enabled by default).
There are too many advanced options now for P checker.

Reply:
Lets do this by default.